src/Features/Blockcore.Features.Miner/UI/NavItem.cs

```diff
@@ -7,11 +7,11 @@ namespace Blockcore.Features.Miner.UI
     public class MineNavigationItem : INavigationItem
     {
         private readonly Network network;
-        private readonly IWalletSyncManager walletSyncManager;
-        public MineNavigationItem(Network network, IWalletSyncManager walletSyncManager)
+        private readonly IWalletManager WalletManager;
```

Comment:
Fields should start with lowercase initial character, properties with uppercase.